sdcm/nemesis.py

```diff
@@ -746,6 +747,18 @@ def disrupt_validate_hh_short_downtime(self):
         assert self.tester.hints_sending_in_progress() is False, "Hints are sent too slow"
         self.tester.verify_no_drops_and_errors(starting_from=start_time)
 
+    def disrupt_snapshot_operations(self):
```

Comment:
need to run `self._set_current_disruption` at the beginnig

Reply:
Done